lib/state/ui/reducer.ts

```diff
@@ -90,9 +90,22 @@ const dialogs: A.Reducer<T.DialogType[]> = (state = [], action) => {
   switch (action.type) {
     case 'CLOSE_DIALOG':
       return state.slice(0, -1);
-
+    case 'TRASH_TAG':
+      return state.filter(
+        (element) => element.type !== 'TRASH-TAG-CONFIRMATION'
```

Comment:
at this point I believe we are dealing with `dialog`s - might be more helpful than the generic "element"

Reply:
Very true. I'll update this.